pkg/exchange/max/exchange.go

```diff
@@ -1134,6 +1134,36 @@ func (e *Exchange) QueryKLines(
 	return kLines, nil
 }
 
+func (e *Exchange) QueryDepth(ctx context.Context, symbol string, limit int) (snapshot types.SliceOrderBook, finalUpdateID int64, err error) {
+	req := e.v3client.NewGetDepthRequest()
+	req.Market(symbol)
```

Comment:
should we sort by price?

Reply:
default it's true, I think both options are fine ?